### PR TITLE
chore: use access url from env resource pool

### PR DESCRIPTION
**Pull request by cmoog**
cc @f0ssel 

[ch5790]

### CODE REVIEW

**Comment by cmoog** [comment]
Waiting to test properly when `AccessURL` is available.

**Comment by cmoog** [comment]
Oh there is an issue with the sshAvailable check, that needs to be on a per resource pool basis.

**Comment by f0ssel** [comment]
Looking good so far, hmu when you got the ssh fix in

**Comment by cmoog** [comment]
@f0ssel it's good to go 👍

**Review by f0ssel — APPROVED** [review]

File: internal/cmd/configssh.go [hunk 1]
```diff
@@ -103,20 +102,17 @@ func configSSH(configpath *string, remove *bool) func(cmd *cobra.Command, _ []st
...
 
-		if !sshAvailable(envs) {
-			return xerrors.New("SSH is disabled or not available for any environments in your Coder Enterprise deployment.")
-		}
-
-		err = canConnectSSH(ctx)
+		envsWithPools, err := coderutil.EnvsWithPool(ctx, client, envs)
 		if err != nil {
-			return xerrors.Errorf("check if SSH is available: unable to connect to SSH endpoint: %w", err)
+			return xerrors.Errorf("resolve env pools: %w", err)
```
Comment (deansheather) [inline internal/cmd/configssh.go:107]: Should we keep canConnectSSH?

**Review by deansheather — APPROVED** [review]

**Comment by deansheather** [comment]
oh too late

File: internal/cmd/configssh.go (hunk 1 above)
Comment (cmoog) [reply internal/cmd/configssh.go:107]: It was kinda dumb anyway....

**Pull request merged**